dev/doc/INSTALL.make.md

```diff
@@ -119,6 +119,19 @@ please see the [contributing guide](../../CONTRIBUTING.md).
 
         make clean
 
+Notes for packagers
+-------------------
+
+The `make install` target for Coq's OCaml parts will call `dune
+install` internally. Before Dune 2.9, `dune install` didn't support
+configuring some particular install paths such as `-docdir` and
+`-configdir`, thus these configure options are ignored by default.
+
+If using Dune 2.9, we recommend patching `Makefile.install` so these
+options are taken into account, if using Dune < 2.8, you may have to
+post-process your package to fix install locations. See
+`Makefile.install` `install-dune` target for more information.
```

Comment:
```suggestion
The `make install` target for Coq's OCaml parts calls `dune
install` internally. Before Dune 2.9, `dune install` didn't support
configuring some installation paths such as `-docdir` and
`-configdir`, thus these configure options were ignored by default.

For Dune 2.9, we recommend patching `Makefile.install` so these
options are taken into account.  For Dune < 2.8, you may have to
post-process your package to fix installation locations. See
`Makefile.install` `install-dune` target for more information.
```

Reply:
Should `< 2.8` be `<= 2.8`? The former doesn't seem consistent with the rest.

Reply:
Or just `< 2.9` or `before 2.9`, requiring involvement of slightly few neurons?  I suppose the condition applies equally to `2.8.2` and `2.8.0`.